.github/CODEOWNERS

```diff
@@ -5,14 +5,14 @@
 /extensions/arc/ @swells
```

Comment:
We might need to do more clean-up.
<img width="300"  alt="image" src="https://github.com/user-attachments/assets/2de3bd8a-f967-49cb-be51-3dc85f3c5121" />